test/functional/job/job_spec.lua

```diff
@@ -119,7 +119,7 @@ describe('jobs', function()
     eq({'notification', 'exit', {0, 0}}, next_msg())
   end)
 
-  it('wont allow jobsend with a job that closed stdin', function()
+  it('won\'t allow jobsend with a job that closed stdin', function()
```

Comment:
You could instead surround the string with double quotes; same goes for the other instances.